README.rst

```diff
@@ -16,17 +16,24 @@ Installation
 redis-py requires a running Redis server. See `Redis's quickstart
 <https://redis.io/topics/quickstart>`_ for installation instructions.
 
+redis-py can be installed using `pip` similar to other Python packages. Do not use `sudo`
+with `pip`. It is usually good to work in a
+[virtualenv](https://virtualenv.pypa.io/en/latest/) or
+[venv](https://docs.python.org/3/library/venv.html) to avoid conflicts with other package
+managers and Python projects. For a quick introduction see
+[Python Virtual Environments in Five Minutes](bit.ly/py-env).
+
```

Comment:
Sorry, I was lazy and wrote that in GitHub comment markdown, not RST...  just a minute and I'll translate what you've got here.

Reply:
```rst
redis-py can be installed using ``pip`` similar to other Python packages. Do not use ``sudo``
with ``pip``. It is usually good to work in a `virtualenv`_ or `venv`_ to avoid conflicts
with other package managers and Python projects. For a quick introduction see
`Python Virtual Environments in Five Minutes`_.

.. _`virtualenv`: https://virtualenv.pypa.io/en/latest/
.. _`venv`: https://docs.python.org/3/library/venv.html
.. _`Python Virtual Environments in Five Minutes`: https://bit.ly/py-env
```

Reply:
#facepalm :) Should have seen that coming myself